### PR TITLE
Search for edit-config and config-file tags and show relevant information

### PR DESCRIPTION
**Pull request by jcesarmobile**
If the plugin has edit-config or config-file tags, check if the Info.plist or AndroidManifest.xml have the required entries already, and if they don't have them, log a relevant message.

### CODE REVIEW

File: cli/src/cordova.ts [hunk 1]
```diff
@@ -179,4 +179,88 @@ export async function copyCordovaJSFiles(config: Config, platform: string) {
...
+}
+
+async function logiOSPlist (configElement: any, config: Config, plugin: Plugin) {
+  const plistPath = resolve(config.ios.platformDir, config.ios.nativeProjectName, config.ios.nativeProjectName, 'Info.plist');
+  const xmlMeta = await readXML(plistPath);
+  const dict = xmlMeta.plist.dict.pop();
+  if (!dict.key.includes(configElement.$.parent)) {
+    let xml = await buildConfigFileXml(configElement);
+    xml = `<key>${configElement.$.parent}</key>${getConfigFileTagContent(xml)}`;
+    console.log(`plugin ${plugin.id} requies to add \n  ${xml} to your Info.plist to work`);
```
Comment (janpio) [inline cli/src/cordova.ts]: typo "requies"

File: cli/src/cordova.ts [hunk 2]
```diff
@@ -179,4 +179,88 @@ export async function copyCordovaJSFiles(config: Config, platform: string) {
...
+      element = xmlElement[0][k];
+    }
+    await Promise.all(configElement[k].map(async (e: any) => {
+      if (element === undefined || !contains(element, e)) {
+        xmlEntries.push(await buildXmlElement(e, k));
+      }
+    }));
+  }));
+  if (xmlEntries.length > 0) {
+    console.log(`plugin ${plugin.id} requies to add \n  ${xmlEntries.join("\n  ")} \nto your AndroidManifest.xml at ${configElement.$.parent.split("/").filter((part: string) => part !== '').join(" ")} level to work\n`);
```
Comment (janpio) [inline cli/src/cordova.ts]: same typo

**Comment by janpio** [comment]
nice solution for a tricky problem.

**Comment by jcesarmobile** [comment]
updated, thanks!

**Pull request merged**